src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -598,6 +597,10 @@ public void afterSave(@NonNull SCMNavigatorOwner owner) {
         }
     }
 
+    GitHubWebHook getHook() {
```

Comment:
The event tests that are not using mockito... those seemed ok. everything else is not interesting to my eyes